charts/apiclarity/values.yaml

```diff
@@ -90,21 +90,26 @@ apiclarity:
 apiclarity-postgresql:
   enabled: true
 
-  ## ConfigMap with scripts to be run at first boot
-  ## NOTE: This will override initdbScripts
-  # initdbScriptsConfigMap:
-
-  ## Secret with scripts to be run at first boot (in case it contains sensitive information)
-  ## NOTE: This can work along initdbScripts or initdbScriptsConfigMap
-  # initdbScriptsSecret:
-
-  ## Specify the PostgreSQL username and password to execute the initdb scripts
-  # initdbUser:
-  # initdbPassword:
+  image:
+  ## Use specified image tag
+    # tag: 14.4.0-debian-11-r4
```

Comment:
maybe better to keep it uncomment?

Reply:
The image should be commented out as well.